signer/src/transaction_signer.rs

```diff
@@ -1148,24 +1162,25 @@ where
 
                 // Perform verification of the signature.
                 tracing::info!("🔐 verifying that the signature can be used to spend a UTXO locked by the new aggregate key");
+                let db = self.context.get_storage_mut();
                 let signature: TaprootSignature = sig.into();
-                mock_tx
-                    .verify_signature(&signature)
-                    .inspect_err(|e| tracing::warn!(?e, "🔐 signature verification failed"))?;
-                tracing::info!("🔐 \x1b[1;32msignature verification successful\x1b[0m");
-
-                self.context
-                    .get_storage_mut()
-                    .verify_dkg_shares(aggregate_key)
-                    .await?;
-                tracing::info!(
-                    "🔐 DKG shares entry has been marked as verified; it is now able to be used"
-                );
+                match mock_tx.verify_signature(&signature) {
+                    Ok(()) => {
+                        tracing::info!("🔐 \x1b[1;32msignature verification successful\x1b[0m");
```

Comment:
nit: remove colors?

Reply:
Done in https://github.com/stacks-network/sbtc/pull/1318/commits/402f13ca3827c551bd814b09261ae1cf5578886b